az-digital/az_quickstart
# Closes #2702: Add unpublished content layout to Marketing Cloud

## Description
When rendering a Marketing Cloud email, if I insert a URL to an unpublished item, I want to have the rendered "Page not found" message to be specific to the layout.

## Related issues
Closes #2702

## How to test
1. Enable the `Quickstart for Marketing Cloud` module
2. Unpublish a news and/or event item
3. View the `Content for email` content page
4. Test the rendered links to ensure they show the unpublished content when not logged in

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
